api/repositories/service_binding_repository.go

```diff
@@ -253,23 +257,65 @@ func (r *ServiceBindingRepo) DeleteServiceBinding(ctx context.Context, authInfo
 }
 
 func (r *ServiceBindingRepo) GetServiceBinding(ctx context.Context, authInfo authorization.Info, guid string) (ServiceBindingRecord, error) {
-	ns, err := r.namespaceRetriever.NamespaceFor(ctx, guid, ServiceBindingResourceType)
+	binding, err := r.getServiceBinding(ctx, authInfo, guid)
 	if err != nil {
-		return ServiceBindingRecord{}, err
+		return ServiceBindingRecord{}, fmt.Errorf("get-service-binding failed to create user client: %w", err)
```

Comment:
This error is already wrapped, no need to wrap again, furthermore the description is incorrect.